F2.txt

```diff
@@ -0,0 +1 @@
+file F2
```

Comment:
What is the purpose of this line, this is invalid code.

Reply:
This is only for testing purpose.